### ISSUE TITLE
I compiled this repo with prod/aot, and ran this demo, "Load Modue 1/2" not work

### ISSUE THREAD

**Issue by rdkmaster** [AndreiVajnaII/lazy-load-ng-modules#1]
I start this repo with this command `ng s --aot --prod --source-map`, this should compile with prod/aot to simulate the production mode.

The "Load component" button works as expected as when ran without `--aot --prod` compilation args. But the "Load module 1/2" buttons not work, they work fine when I ran without `--aot --prod`.

This is what I see.

![aa](https://user-images.githubusercontent.com/16191626/80351862-97717d00-88a5-11ea-895c-a91188eec8df.gif)

**Comment by AndreiVajnaII** [comment]
Indeed, this is an issue with Angular. It does not work when build with --prod. I'm not sure if it's intentional or not. [See here the brief discussion](https://github.com/angular/angular/issues/36255).
